src/Functions/multiIf.cpp

```diff
@@ -373,18 +385,115 @@ class FunctionMultiIf final : public FunctionIfBase
         }
     }
 
+    template<typename S>
+    static void calculateWhetherUseMemoryCopy(PaddedPODArray<S> & inserts, std::vector<Instruction> & instructions, std::optional<int> & instruction_use_memory_copy,
+        double threshold_use_memcpy)
+    {
+        if (!instruction_use_memory_copy.has_value())
+        {
+            std::vector<UInt64> instruction_insert_sizes(instructions.size());
```

Comment:
Initialize all elements in vector to zero. 

instruction_insert_sizes(instructions.size(), 0);